downloader_app/tests/test_tiff_downloader.py

```diff
@@ -2,20 +2,24 @@
 import sys
 
 import pandas as pd
-import pytest
 import requests
 from netCDF4 import Dataset
 
 from downloader_app import shapefile_module as shpm
 from downloader_app import tiff_downloader as td
-from downloader_app.settings import BASE_DIR
 
-LOCAL = os.path.join(BASE_DIR, "downloader_app")
-sys.path.insert(0, LOCAL)
+BASE_DIR = os.path.dirname(
+    os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+)
+WORK_DIR = os.path.join(BASE_DIR, "downloader_app")
+sys.path.insert(0, WORK_DIR)
+
 
 # Set the filepath and load in a shapefile.
-SHP_PATH = os.path.join(LOCAL, "RJ_Mun97_region/RJ_Mun97_region.shp")
-DOWNLOADFILES_PATH = os.path.join(LOCAL, "DownloadedFiles")
+SHP_PATH = os.path.join(
+    WORK_DIR, "shape_region/RJ_Mun97_region/RJ_Mun97_region.shp"
```

Comment:
if you need a shapefile for test ... add it into a test folder, for example: `myproject_path/myapp_path/tests/data/RJ_Mun97_region.shp`

Reply:
It's done!

Reply:
My Comment from above applies here as well,  do not use shapefiles. Use the Geojson from the database.